modules/planning/tasks/deciders/path_bounds_decider.cc

```diff
@@ -16,19 +16,174 @@
 
 #include "modules/planning/tasks/deciders/path_bounds_decider.h"
 
+#include <algorithm>
+#include <limits>
+#include <string>
+#include <tuple>
+#include <vector>
+
+#include "modules/common/configs/vehicle_config_helper.h"
+
 namespace apollo {
 namespace planning {
 
+using apollo::common::ErrorCode;
 using apollo::common::Status;
+using apollo::common::VehicleConfigHelper;
+
+constexpr double kPathBoundsDeciderHorizon = 100.0;
+constexpr double kPathBoundsDeciderResolution = 0.1;
+constexpr double kDefaultLaneWidth = 5.0;
+constexpr double kDefaultRoadWidth = 20.0;
+constexpr double kRoadEdgeBuffer = 0.2;
```

Comment:
Move the parameters to gflags. Some of the parameters are already in gflags. It is better to merge them other than defining new ones.

Reply:
Yes, I intended to do that once I finish the entire implementation. That way we know what old gflags can be deleted. I just didn't want to add too many gflags without deleting any.

Reply:
OK.